helm/templates/config/configmap.yaml

```diff
@@ -4,70 +4,29 @@ metadata:
-  DJANGO_TIME_ZONE: {{ .Values.env.DJANGO_TIME_ZONE }}
-  DJANGO_DEBUG: {{ .Values.env.DJANGO_DEBUG | quote }}
-  DJANGO_ALLOWED_HOST: {{ .Values.env.DJANGO_ALLOWED_HOST | quote }}
-  APP_HTTP_PROTOCOL: {{ .Values.env.APP_HTTP_PROTOCOL | quote }}
-
-  # Email
-  DEFAULT_FROM_EMAIL: {{ required "env.DEFAULT_FROM_EMAIL" .Values.env.DEFAULT_FROM_EMAIL | quote }}
-  EMAIL_USE_TLS: {{ required "env.EMAIL_USE_TLS" .Values.env.EMAIL_USE_TLS | quote }}
-
-  {{- if .Values.redis.enabled }}
-  # Celery
-  CELERY_REDIS_URL: "redis://{{ printf "%s-master" (include "common.names.fullname" .Subcharts.redis) }}:6379/0"
-  CELERY_BROKER_URL: "redis://{{ printf "%s-master" (include "common.names.fullname" .Subcharts.redis) }}:6379/0"
-  CELERY_RESULT_BACKEND: "redis://{{ printf "%s-master" (include "common.names.fullname" .Subcharts.redis) }}:6379/0"
-  # Cache
-  DJANGO_CACHE_REDIS_URL: "redis://{{ printf "%s-master" (include "common.names.fullname" .Subcharts.redis) }}:6379/1"
-  {{- else }}
-  # Celery
-  CELERY_BROKER_URL: {{ required "env.CELERY_BROKER_URL" .Values.env.CELERY_BROKER_URL | quote }}
-  CACHE_REDIS_URL: {{ required "env.CACHE_REDIS_URL" .Values.env.CACHE_REDIS_URL | quote }}
-  CELERY_RESULT_BACKEND: {{ required "env.CELERY_RESULT_BACKEND" .Values.env.CELERY_RESULT_BACKEND | quote }}
-  # Cache
-  DJANGO_CACHE_REDIS_URL: {{ required "env.DJANGO_CACHE_REDIS_URL" .Values.env.DJANGO_CACHE_REDIS_URL | quote }}
-  {{- end }}
-
-  # Sentry
-  SENTRY_TRACES_SAMPLE_RATE: {{ .Values.env.SENTRY_TRACES_SAMPLE_RATE | quote }}
-  SENTRY_PROFILE_SAMPLE_RATE: {{ .Values.env.SENTRY_PROFILE_SAMPLE_RATE | quote }}
-  
   # Vector Database
-  {{- if .Values.redis.enabled }}
+  {{- if .Values.app.redis.enabled }}
   QDRANT_DB_HOST: {{ include "qdrant.fullname" .Subcharts.qdrant | quote }}
   {{- else }}
-  QDRANT_DB_HOST: {{ required "env.QDRANT_DB_HOST" .Values.env.QDRANT_DB_HOST | quote }}
+  QDRANT_DB_HOST: {{ required "env.QDRANT_DB_HOST" .Values.app.env.QDRANT_DB_HOST | quote }}
   {{- end }}
-  QDRANT_DB_PORT: {{ required "env.QDRANT_DB_PORT" .Values.env.QDRANT_DB_PORT | quote }}
-  QDRANT_DB_COLLECTION_NAME: {{ required "env.QDRANT_DB_COLLECTION_NAME" .Values.env.QDRANT_DB_COLLECTION_NAME | quote }}
+  QDRANT_DB_PORT: {{ required "env.QDRANT_DB_PORT" .Values.app.env.QDRANT_DB_PORT | quote }}
+  QDRANT_DB_COLLECTION_NAME: {{ required "env.QDRANT_DB_COLLECTION_NAME" .Values.app.env.QDRANT_DB_COLLECTION_NAME | quote }}
 
   # Embedding Model
-  {{- if .Values.embeddingModel.enabled }}
+  {{- if .Values.app.embeddingModel.enabled }}
   EMBEDDING_MODEL_URL: http://{{ template "tc-chatbot-backend.fullname" . }}-embedding-model-svc
   {{- else }}
-  EMBEDDING_MODEL_URL: {{ required ".Values.env.EMBEDDING_MODEL_URL" .Values.env.EMBEDDING_MODEL_URL | quote }}
+  EMBEDDING_MODEL_URL: {{ required ".Values.env.EMBEDDING_MODEL_URL" .Values.app.env.EMBEDDING_MODEL_URL | quote }}
   {{- end }}
```

Comment:
We don't need to check, use can just use the env.

Make sure to treat the env as function

Something like this
https://github.com/toggle-corp/charts/blob/main/toggle/django-app/templates/config/configmap.yaml#L12C16-L12C42